src/QSymbolicsBase/express.jl

```diff
@@ -37,8 +37,6 @@ julia> express(QuantumSymbolics.X, CliffordRepr(), UseAsObservable())
 + X
 ```
 """
-function express end
```

Comment:
Could you update the docstring above, so it is clear this is about symbolics, not about all of the express capabilities? Just adding the work "symbolic" somewhere would be enough.